<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="mainwid">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1301</width>
      <height>581</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Homie">
     <widget class="QLabel" name="lblhome">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>701</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>The Homie</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnsub1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>170</y>
        <width>721</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Subject 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnsub2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>721</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Subject 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chapter">
     <widget class="QLabel" name="lblchap">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>701</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Subject-   Chapters</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchp1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Chapter-1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchp2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>211</x>
        <y>250</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Chapter-2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchp3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>211</x>
        <y>340</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Chapter-3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchp1_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>211</x>
        <y>420</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchp1_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>340</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchp1_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>160</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchp1_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>250</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblsubno_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Questions">
     <widget class="QLabel" name="lblhome_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Subject-   Chapter(   )</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblquestion">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>92</y>
        <width>781</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btna">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>90</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:3px solid white;border-radius:35;</string>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnb">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>92</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:3px solid white;border-radius:35;</string>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnc">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>180</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:3px solid white;
border-radius:35%;</string>
      </property>
      <property name="text">
       <string>c</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnd">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>180</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:3px solid white;border-radius:35;</string>
      </property>
      <property name="text">
       <string>d</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnans">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>270</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:3px solid white;border-radius:35;</string>
      </property>
      <property name="text">
       <string>Answer</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbloutput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>1311</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnnxt">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>270</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:3px solid white;border-radius:35;</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblcurrent">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>20</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblcurrent_2">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>20</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/10</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblchpno">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblsubno">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>18</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="markout">
     <widget class="QLabel" name="lblcongrat">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>861</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CONGRATULATION!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>1221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string>You got    marks out of 10.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblmark">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>250</y>
        <width>61</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>420</y>
        <width>631</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 5px solid rgb(0, 170, 127);
background-color: rgb(255, 255, 255);
color: rgb(0, 170, 127);
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string>Go to Homie</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logo and title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>Go to Homie</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
